common/setups/rasr/gmm_system.py

```diff
@@ -1447,9 +1447,10 @@ def run(self, steps: Union[List[str], RasrSteps]):
 
             # ---------- Forced Alignment ----------
             if step_name.startswith("forced_align"):
-                for trn_c in self.train_corpora:
+                corpus_keys = step_args.pop("corpus_keys", [])
```

Comment:
should the default not stay `self.train_corpora`?
```suggestion
                corpus_keys = step_args.pop("corpus_keys", self.train_corpora)
```

Reply:
nvm this already is a list, somehow thought this was just a corpus, then this would have worked ofc.